generate_hierarchical_cv_db/__main__.py

```diff
@@ -0,0 +1,81 @@
+from cv_library.compressor import Compressor
```

Comment:
I'd actuall prefer this not to be a module, but a runnable file `generate_hierarchical_cv_db.py` in the root